go/vaas/pkg/vaas/vaas_interface.go

```diff
@@ -0,0 +1,15 @@
+package vaas
+
+import (
+	msg "vaas/pkg/messages"
+)
+
+type IVaaS interface {
```

Comment:
If you have only one type, implementing your interface, it is completely fine to put them in the same file